schema/Contact/ACLContactCache.entityType.php

```diff
@@ -61,5 +67,14 @@
         'callback' => ['CRM_ACL_BAO_ACL', 'operation'],
       ],
     ],
+    'domain_id' => [
+      'title'  => ts('Domain'),
+      'sql_type' => 'int unsigned',
+      'input_type' => 'Number',
+      'description' => ts('Implicit FK to civicrm_domain'),
```

Comment:
Why only an implicit FK? Why not make it a real FK with `ON DELETE CASCADE`?

Reply:
Given its a cache table I think its better to not have direct foreign keys see also how there is no FK with the civicrm_contact table for either user_id or contact_id

Reply:
Ok fair enough